### ISSUE TITLE
Are you still planning to release the training code?

### ISSUE THREAD

**Issue by Mark36579** [buaacyw/MeshAnythingV2#14]
If you guys don't want to open source, there is no need to claim "open-source to the community" as one of your contribution in the paper. I noticed that the V1 version of MeshAnything, despite that it has been out months ago, still has no sign on updating the training script to the repo. 

So if this is not a scam and you are not just cherry-picking some unicorn hyperparameters, please release the training code. Otherwise, please update your paper since you cannot claim that you "open-source" to the community. Thanks.

**Comment by Mrguanglei** [comment]
@Mark36579   Yes, this behavior is really sad. I waited a long time to train the code. If you have the training code, you can let me know. Thanks

**Comment by buaacyw** [comment]
Sadly, MeshAnything is rejected. 

We plan to release the training code once paper is accepted.

**Comment by GZY219** [comment]
Anyone who already finished the training code want to share his or her code?
